<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CInternalsComponent</class>
 <widget class="QWidget" name="CInternalsComponent">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>277</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="vl_InternalsComponent">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tw_Internals">
     <property name="currentIndex">
      <number>2</number>
     </property>
     <widget class="QWidget" name="tb_Debug">
      <attribute name="title">
       <string>Debug</string>
      </attribute>
      <layout class="QVBoxLayout" name="vl_Debug">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>3</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QGroupBox" name="gb_DebugContext">
         <property name="title">
          <string>Contexts</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QCheckBox" name="cb_DebugContextAudio">
            <property name="text">
             <string>Audio</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QCheckBox" name="cb_DebugContextNetwork">
            <property name="text">
             <string>Network</string>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QCheckBox" name="cb_DebugContextOwnAircraft">
            <property name="text">
             <string>Own aircraftt</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QCheckBox" name="cb_DebugContextSimulator">
            <property name="text">
             <string>Simulator</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QCheckBox" name="cb_DebugContextApplication">
            <property name="text">
             <string>Application</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="gb_DebugDriverAndInterpolator">
         <property name="title">
          <string>Driver plugin and interpolator</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <widget class="QCheckBox" name="cb_DebugDriver">
            <property name="text">
             <string>Driver</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QCheckBox" name="cb_DebugInterpolator">
            <property name="text">
             <string>Interpolator</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lbl_Disclaimer_1">
         <property name="text">
          <string>WORK IN PROGRES!!!!!</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="vs_DebugWidget">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tb_Statistics">
      <attribute name="title">
       <string>Statistics</string>
      </attribute>
      <widget class="QLabel" name="lbl_Disclaimer_2">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>60</y>
         <width>267</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>WORK IN PROGRES!!!!!</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tb_AircraftParts">
      <attribute name="title">
       <string>Aircraft parts</string>
      </attribute>
      <layout class="QVBoxLayout" name="vl_AircraftParts">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QGroupBox" name="gb_Aircraft">
         <property name="title">
          <string>Aircraft</string>
         </property>
         <layout class="QVBoxLayout" name="vl_GroupAircraftSelector">
          <property name="spacing">
           <number>2</number>
          </property>
          <property name="leftMargin">
           <number>4</number>
          </property>
          <property name="topMargin">
           <number>6</number>
          </property>
          <property name="rightMargin">
           <number>4</number>
          </property>
          <property name="bottomMargin">
           <number>2</number>
          </property>
          <item>
           <widget class="BlackGui::Components::CRemoteAircraftSelector" name="comp_RemoteAircraftSelector">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="fr_AircraftParts">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="vl_AircraftPartsSplitterFrame">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QSplitter" name="sp_AircraftParts">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <widget class="QGroupBox" name="gb_AircraftParts">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="title">
              <string>Parts</string>
             </property>
             <layout class="QVBoxLayout" name="vl_GroupParts">
              <property name="spacing">
               <number>6</number>
              </property>
              <property name="leftMargin">
               <number>4</number>
              </property>
              <property name="topMargin">
               <number>6</number>
              </property>
              <property name="rightMargin">
               <number>4</number>
              </property>
              <property name="bottomMargin">
               <number>2</number>
              </property>
              <item>
               <layout class="QGridLayout" name="gl_AircraftParts">
                <property name="spacing">
                 <number>6</number>
                </property>
                <item row="3" column="2">
                 <widget class="QCheckBox" name="cb_AircraftPartsLightsLanding">
                  <property name="text">
                   <string>Landing</string>
                  </property>
                 </widget>
                </item>
                <item row="4" column="1">
                 <widget class="QCheckBox" name="cb_AircraftPartsLightsStrobe">
                  <property name="text">
                   <string>Strobe</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0">
                 <widget class="QCheckBox" name="cb_AircraftPartsGearDown">
                  <property name="text">
                   <string>Gear down</string>
                  </property>
                 </widget>
                </item>
                <item row="6" column="1">
                 <widget class="QCheckBox" name="cb_AircraftPartsEngine2">
                  <property name="text">
                   <string>2</string>
                  </property>
                 </widget>
                </item>
                <item row="5" column="0">
                 <widget class="QLabel" name="lbl_AircraftPartsEngines">
                  <property name="text">
                   <string>Engines:</string>
                  </property>
                 </widget>
                </item>
                <item row="6" column="0">
                 <widget class="QCheckBox" name="cb_AircraftPartsEngine1">
                  <property name="text">
                   <string>1</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="0">
                 <widget class="QLabel" name="lbl_AircraftPartsFlapsPercentage">
                  <property name="text">
                   <string>Flaps %:</string>
                  </property>
                 </widget>
                </item>
                <item row="7" column="0">
                 <widget class="QCheckBox" name="cb_AircraftPartsEngine4">
                  <property name="text">
                   <string>4</string>
                  </property>
                 </widget>
                </item>
                <item row="6" column="2">
                 <widget class="QCheckBox" name="cb_AircraftPartsEngine3">
                  <property name="text">
                   <string>3</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1">
                 <widget class="QCheckBox" name="cb_AircraftPartsSpoilers">
                  <property name="text">
                   <string>Spoilers</string>
                  </property>
                 </widget>
                </item>
                <item row="4" column="0">
                 <widget class="QCheckBox" name="cb_AircraftPartsLightsBeacon">
                  <property name="text">
                   <string>Beacon</string>
                  </property>
                 </widget>
                </item>
                <item row="3" column="1">
                 <widget class="QCheckBox" name="cb_AircraftPartsLightsNav">
                  <property name="text">
                   <string>Nav</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="0">
                 <widget class="QLabel" name="lbl_AircraftPartsLights">
                  <property name="text">
                   <string>Lights:</string>
                  </property>
                 </widget>
                </item>
                <item row="3" column="0">
                 <widget class="QCheckBox" name="cb_AircraftPartsLightsTaxi">
                  <property name="text">
                   <string>Taxi</string>
                  </property>
                 </widget>
                </item>
                <item row="4" column="2">
                 <widget class="QCheckBox" name="cb_AircraftPartsLightsLogo">
                  <property name="text">
                   <string>Logo</string>
                  </property>
                 </widget>
                </item>
                <item row="7" column="1">
                 <widget class="QCheckBox" name="cb_AircraftPartsEngine5">
                  <property name="text">
                   <string>5</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="QSpinBox" name="sb_AircraftPartsFlapsPercentage">
                  <property name="maximum">
                   <number>100</number>
                  </property>
                  <property name="singleStep">
                   <number>5</number>
                  </property>
                 </widget>
                </item>
                <item row="2" column="1">
                 <widget class="QPushButton" name="pb_AircraftPartsLightsOn">
                  <property name="text">
                   <string>all on</string>
                  </property>
                 </widget>
                </item>
                <item row="5" column="1">
                 <widget class="QPushButton" name="pb_AircraftPartsEnginesOn">
                  <property name="text">
                   <string>all on</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="2">
                 <widget class="QCheckBox" name="cb_AircraftPartsIsOnGround">
                  <property name="text">
                   <string>on ground</string>
                  </property>
                 </widget>
                </item>
                <item row="7" column="2">
                 <widget class="QCheckBox" name="cb_AircraftPartsEngine6">
                  <property name="text">
                   <string>6</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="2">
                 <widget class="QPushButton" name="pb_AircraftPartsLightsOff">
                  <property name="text">
                   <string>all off</string>
                  </property>
                 </widget>
                </item>
                <item row="5" column="2">
                 <widget class="QPushButton" name="pb_AircraftPartsEnginesOff">
                  <property name="text">
                   <string>all off</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="2">
                 <widget class="QPushButton" name="pb_AircraftPartsUiToJson">
                  <property name="text">
                   <string>to JSON</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
            <widget class="QTextEdit" name="te_AircraftPartsJson"/>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="qw_AircraftPartsSendButtons" native="true">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <layout class="QHBoxLayout" name="hl_AircraftPartsSendButtons">
          <property name="spacing">
           <number>6</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QCheckBox" name="cb_AircraftPartsIncremental">
            <property name="text">
             <string>Incremental</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pb_SendAircraftPartsJson">
            <property name="text">
             <string>Send JSON</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pb_SendAircraftPartsGui">
            <property name="text">
             <string>Send GUI</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>BlackGui::Components::CRemoteAircraftSelector</class>
   <extends>QFrame</extends>
   <header>blackgui/components/remoteaircraftselector.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
